<ui version="4.0">
 <author/>
 <comment/>
 <exportmacro/>
 <class>DialogAddValueToReplace</class>
 <widget class="QDialog" name="DialogAddValueToReplace">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Value To Replace</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="labelMarketplaceId">
       <property name="text">
        <string>Marketplace ID:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEditMarketplaceId"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelCountryCode">
       <property name="text">
        <string>Country Code:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEditCountryCode"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelLangCode">
       <property name="text">
        <string>Language Code:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEditLangCode"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelAttributeId">
       <property name="text">
        <string>Attribute ID:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEditAttributeId"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="labelValueFrom">
       <property name="text">
        <string>Value To Replace:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="lineEditValueFrom"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="labelValueTo">
       <property name="text">
        <string>Replacement Value:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="lineEditValueTo"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <pixmapfunction/>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogAddValueToReplace</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogAddValueToReplace</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
